meta-mbl-distro/recipes-core/mbl-initramfs-init/files/initramfs-init-script.sh

```diff
@@ -27,6 +30,12 @@ then
     exec </dev/console >/dev/console 2>/dev/console
 fi
 
+# Initialise the hardware watchdog and set the timeout. We need to do this
+# in the initramfs, as the watchdog needs to be started before switching to the rootfs.
+printf "Setting hardware watchdog with device filename %s to timeout of %s seconds\n" \
+    "${MBL_WATCHDOG_DEVICE_FILENAME}" "${MBL_WATCHDOG_TIMEOUT_SECS}"
+mbl-watchdog-init --timeout ${MBL_WATCHDOG_TIMEOUT_SECS} --device ${MBL_WATCHDOG_DEVICE_FILENAME}
```

Comment:
I suggest double quotes around `${MBL_WATCHDOG_TIMEOUT_SECS}` and `${MBL_WATCHDOG_DEVICE_FILENAME}`, even if only for consistency with the previous line (I don't see why they would need quoting in the `printf` line but not this one).